src/utils/getReactNativeVersion.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+
+const path = require('path');
+const fs = require('fs');
+const logger = require('../logger');
+
+/**
+* Search top-most node_modules for React Native
+* and return it's version
+*/
+
+module.exports = (): string => {
+  const pckJsonLocation = path.join(
```

Comment:
`require.resolve('react-native/package.json')` is probably better

Reply:
I'd insist on using `cwd` to get it

Reply:
I'd suggest using https://github.com/sindresorhus/resolve-cwd

Reply:
If you insist :)